src/plugins/share/common/config.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
```

Comment:
__question:__ @elastic/kibana-core do we support schemas produced by `@kbn/config-schema` on the client side now? If not, I'm wondering if it'd be more correct to move schema from `common` to `server` instead.

Reply:
Apparently `joi` is usable on the browser (the source of the original issue) since `16.0.0` and we are on 17.7.1. At least that is what this repo indicates https://github.com/jeffbski/joi-browser

Reply:
Aha, nice! Then feel free to ignore my comment if you need schema on both client and server sides.